src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -237,7 +266,7 @@ private void iterateOverAttributes(BiConsumer<String, String> attribute, XMLStre
     }
 
     private boolean allElementHandled() {
-        return this.jobHandled && this.genericInformationHandled && this.variablesHandled;
+        return this.jobHandled && this.genericInformationHandled && this.variablesHandled && this.descriptionHandled;
     }
```

Comment:
I would make isDecriptionHandler() method

Reply:
descriptionHandled is needed only for private method. If create isDecriptionHandler() method it will be private method as well. So it's not useful to my mind. Do u agree?

Reply:
There is nice The Uniform Access Principle. It is implemented in Scala.